content/ADR-125-realm-modifiers-for-worlds-in-renderer.md

```diff
@@ -0,0 +1,196 @@
+  configurations: {
+    networkId: number
+    realmName?: string
+    scenesUrn: string[]
+    globalScenesUrn: string[]
+  }
+  content: {
+    healthy: boolean
+    version?: string
+    commitHash?: string
+    publicUrl: string
+  }
+  comms: {
+    healthy: boolean
+    protocol: string
+    fixedAdapter?: string
+    usersCount?: number
+  }
+  lambdas: {
+    healthy: boolean
+    version?: string
+    commitHash?: string
+    publicUrl: string
+  }
+  bff?: {
+    healthy: boolean
+    userCount: number
+    commitHash?: string
+    publicUrl: string
+  }
+}
+```
+
+Example:
+
+```json
+{
+  "healthy": true,
+  "content": {
+    "healthy": true,
+    "version": "5.1.2",
+    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
+    "publicUrl": "https://peer.decentraland.org/content"
+  },
+  "configurations": {
+    "realmName": "catalyst-4",
+    "networkId": 1
+  },
+  "comms": {
+    "healthy": true,
+    "protocol": "v3",
+    "fixedAdapter": "ws-room:mini-comms.decentraland.org/rooms/test-room"
+  },
+  "lambdas": {
+    "healthy": true,
+    "version": "5.1.2",
+    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
+    "publicUrl": "https://peer.decentraland.org/lambdas"
+  },
+  "bff": {
+    "healthy": true,
+    "commitHash": "369c5dafeda62a1b16f5232cd477565cc3f3d513",
+    "userCount": 0,
+    "publicUrl": "/"
+  }
+}
+```
+
+## Current Implementation 
+
+![Untitled Diagram drawio](https://user-images.githubusercontent.com/1999557/202185605-65ebf318-a2bd-40c9-8908-200daa2166b5.png)
+
+
+## Task breakdown for Explorer
+
+To complete the Beta version for worlds, the following modifiers are required:
+
+- **Skybox Modifier (V1 task):** Create a skybox modifier to allow the world creator control its state.
+	- Skybox settings (time and dynamic toggle) will be unavaible while in world; they should be non-interactable.
+	- A warning message should be added in settings to explicit to the user that the skybox is being controller by the creator
+	- A specific time should be added to the AboutResponse.Configuration to allow the creators set the time of their World
+	- Skybox should be set accordingly to the new variable mentioned above
+
+- **Realms Blocker Modifier:** Extra functionality should be added to the current green blockers modifier.
+	- Green blockers should appear 2 parcels away from the scene area
+
+- **Loading Screen Modifier (V1 task):** This are required functionalities that should be built on top of the current loading screen when accessing worlds. Requires coordination with the design team.
+	- A message of entering world should be added
+	- If a world is full (100 users), a warning message should appear (“The world you are trying to access is full. Please try again later”)
+	- Add an image url for the loading screen to the AboutResponse json
+	- If an image url exists, set it as background
+
+- **Access World Modifier**
+	- If user tries to go to an unexistent world, an empty world should be loaded
```

Comment:
Not sure how would this work... is this part of the about response? If it is, then the world does exist already. Otherwise the world content server should have 404'ed instead of returning a about response.

Reply:
@pentreathm @menduz

Reply:
Agreed, there is no such thing as an empty world unless it is an empty scene deployed to a NAME in the world server. Otherwise, if it doesn't exists, it should be managed by the client.

Reply:
Can we have a configurable "fallback-world.dcl.eth" which can be deployed under the same rules as the rest? It will reduce one twist of configuration of the client and will enable common tools (cli, worlds server) to configure all the aspects of that empty parcel without requiring a release of servers or client itself. Wdyt?

Reply:
I think it would be possible but as a user, if I type wrong the name of a world, I would expect the client to tell me that the world doesn't exist instead of taking me to an empty scene or we can make the fallback scene something fun and let users know that they are in a 404 World

Reply:
Oh yeah, I love that approach, but that conflicts with the product definition. I definitely prefer the 404.

Reply:
I love that approach too. @olavra ?

Reply:
I love the idea of a 404 fallback scenes that we can build in the content team, and do crazy stuff. For the scope of the beta, I suggest to stick with the decentraland.org/404 page